web/src/scheduled_messages_feed_ui.ts

```diff
@@ -33,13 +35,17 @@ function get_scheduled_messages_matching_narrow() {
                 return true;
             }
         } else if (scheduled_message.type === "stream") {
-            if (narrow_state.stream_sub() === undefined) {
+            if (narrow_state.stream_sub() === undefined || narrow_state.topic() === undefined) {
```

Comment:
Cannot compare if either of the two properties is undefined.

Reply:
Agreed, though maybe you just just fetch these as variables rather than calling these functions twice -- doing so would allow you to avoid the `!` type assertions a couple lines down.

This refactor can be a little prep commit in the same PR.

Reply:
Could `current_stream_sub` and `current_topic` be suitable names for the properties (as they are properties of `narrow`).